packages/sdk/contractkit/src/wrappers/Attestations.ts

```diff
@@ -331,7 +331,11 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
 
         const nameClaim = metadata.findClaim(ClaimTypes.NAME)
 
-        const resp = await fetch(`${attestationServiceURLClaim.url}status`)
+        const resp = await fetch(
+          `${attestationServiceURLClaim.url}${
+            attestationServiceURLClaim.url.substr(-1) === '/' ? '' : '/'
+          }status`
+        )
```

Comment:
Too bad JavaScript doesn't provide a builtin way to join url parts.

We do have a string helper though which does the same as what you did: https://github.com/celo-org/celo-monorepo/blob/4276a18e2f9bb5dfc9ef96695bae72bc67a9215b/packages/sdk/base/src/string.ts#L1-L7

Reply:
Nested deep in the identity package we do https://github.com/celo-org/celo-monorepo/blob/master/packages/sdk/identity/src/offchain/utils.ts#L360.